### ISSUE TITLE
[Bug]: Stremio 5 auto play next episode

### ISSUE THREAD

**Issue by Jarvis636** [Stremio/stremio-bugs#1047]
### Stremio Platform

Desktop App (Windows)

### Describe the bug

Stremio 5 is awesome I love it but the only issue I'm experiencing is the auto play next episode it doesn't skip to the next episode it closes off instead.

### To Reproduce

Stremio 5 is awesome I love it but the only issue I'm experiencing is the auto play next episode it doesn't skip to the next episode it closes off instead.

### Expected behavior

Auto play to the next episode 

### Desktop (if applicable)

Stremio 5

**Comment by PixelQubed** [comment]
Yes this is the only issue I have but its a big one when I'm just watching a series and it fails to play, then I have to go out of my game and fix it.

**Comment by aadnan181** [comment]
I think fixing this issue should be the dev's top priority.

**Comment by vladskz** [comment]
Issue is also present on AndroidTV and GoogleTV  latest build 1.6.12

**Comment by Ninu-Nightwatch** [comment]
Came here to report this.

**Comment by aadnan181** [comment]
> Came here to report this.

I think the devs just fixed this issue. Found this changelog in today's update.

![Image](https://github.com/user-attachments/assets/386e92c4-d59b-4232-86e1-3bc57d10fc7f)

**Comment by TRtomasz** [comment]
hello, it is still not fixed for the desktop app

**Comment by aadnan181** [comment]
> hello, it is still not fixed for the desktop app

Since the web app got fixed. Can we expect the desktop app to get fixed soon?

**Comment by TRtomasz** [comment]
will be fixed in next release

**Comment by bilzeebubb** [comment]
> will be fixed in next release

thank you! is there a time frame for the next release?

**Comment by StripClubDJ** [comment]
Log out and log back in worked on sons android TV 😁
